### PR TITLE
Documentation: Using OpenAPI Generator to Generate

### PR DESCRIPTION
**Pull request by cdoron**
Code for a Data Catalog Connector (in the Go Language)

Signed-off-by: Doron Chen <cdoron@il.ibm.com>

### CODE REVIEW

File: site/docs/contribute/openapi.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Using OpenAPI Generator to Generate Code for a Data Catalog Connector (in the Go Language)
+
+The Fybrik repository contains specification files that detail the data catalog connector API. These files include [datacatalog.spec.yaml](https://github.com/fybrik/fybrik/blob/master/connectors/api/datacatalog.spec.yaml) and [taxonomy.json](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/). They detail all the fields that the data catalog connector should expect for each of the supported operations: createAsset, getAssetInfo, deleteAsset, and updateAsset.
+
```
Comment (revit13) [inline site/docs/contribute/openapi.md:4]: @cdoron @rohithdv maybe add the command line to generate the code with openapi generator as it is currently not documented anywhere.

File: site/docs/contribute/openapi.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+The OpenAPI generator is a tool that can be used to generate the skeleton code for REST servers in a variety of programming languages, given a specification file (written in adherence to the [OpenAPI standard](https://swagger.io/specification/)). In our case, we used the OpenAPI generator to generate skeleton code for a Fybrik data catalog connector server, in the [go](https://go.dev/) programming language. As expected, this skeleton code does not provide any functionality, since the specification file details only the API, not the functionality. Also, the behavior of the actual connector code must surely depend on the data catalog chosen to organize the Fyrbik assets.
+
+Here is a snippet of the Makefile used to automatically generate code:
+
+```
+generate-code:
+        git clone https://github.com/fybrik/fybrik/
+        cd fybrik && git checkout v0.7.0
```
Comment (roytman) [inline site/docs/contribute/openapi.md:12]: the branch/version should be  env variable
Some explanation about env variables is required.

File: site/docs/contribute/openapi.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Using OpenAPI Generator to Generate Code for a Data Catalog Connector (in the Go Language)
```
Comment (roytman) [inline site/docs/contribute/openapi.md:1]: please replace to 
```
# Using OpenAPI Generator to Generate Code for a Data Catalog Connector 
(in the Go Language)
```

File: site/docs/contribute/openapi.md (hunk 2 above)
Comment (cdoron) [reply site/docs/contribute/openapi.md:12]: Replaced v0.7.0 with environment variable.
I think that explaining all the environment variables here would be too long and confusing

**Review by roytman — APPROVED** [review]
I approve it, but can you think about shorter left side menu title?
image

**Comment by cdoron** [comment]
> I approve it, but can you think about a shorter left side menu title? image

How about simply: "Using OpenAPI Generator"  ?

**Review by roytman — APPROVED** [review]

**Pull request merged**